### DISCUSSION TITLE
x-callback-url launch?

### DISCUSSION THREAD

**Discussion by lkonstan** [ttscoff/bunch-docs#288] [Ideas]
I can't find similar discussions elsewhere in the forum, but sorry if I'm replicating another question. Is there a way to launch a Bunch using an x-callback-url? If not, I'd find that useful. I manage a lot of writing projects and keep a Workflowy list of my projects and would love to associate each Workflowy node with a Bunch-specific x-callback-url to activate my workspace configuration for that project.

**Comment by ttscoff** [comment]
Sure, https://bunchapp.co/docs/integration/url-handler/#x-callback-url

-Brett

**Reply by lkonstan** [reply]
Thanks! Sorry I missed that in the documentation.
